test/providers.shared.test.ts

```diff
@@ -93,4 +93,23 @@ describe('maybeLoadFromExternalFile', () => {
 
     cliState.basePath = undefined;
   });
+
+  it('should handle list of paths', () => {
+    const basePath = './relative/path';
+    cliState.basePath = basePath;
+    jest.mocked(fs.readFileSync).mockReturnValue(mockJsonContent);
+
+    maybeLoadFromExternalFile([
+      'file://test1.txt',
+      'file://test2.txt',
+      'file://test3.txt',
+    ]);
```

Comment:
to fix the formatting error in the ci 

```suggestion
    maybeLoadFromExternalFile(['file://test1.txt', 'file://test2.txt', 'file://test3.txt']);
```